onboarding.md

```diff
@@ -0,0 +1,42 @@
+## Welcome to SIG-Security team 👋
+
+## Onboarding Tasks
+
+Alright! Time to get our hands dirty 👷 Work through and get acquainted with the process. 
+<br>Please reach out to the team if you need any support 😃
+
+## Lets get going!
+As you walk through this, please remember that it is a living checklist and may require updates or corrections. You can propose updates to the template by editing it under [Onboarding](onboarding.md). 
+
+- [ ] Enable 2FA on GitHub
+- [ ] Review the [Code of Conduct](CODE_OF_CONDUCT.md)
+- [ ] Skim through [Contributing to Eclipse Tractus-X](CONTRIBUTING.md)
+- [ ] Create an Eclipse Foundation Account - [Sign Up](https://accounts.eclipse.org/user/register)
```

Comment:
I think if you just add the [Tractus-X Eclipse Foundation Rules](https://eclipse-tractusx.github.io/docs/oss/getting-started) bullet point, you can skip some of the others, since they are already explained on the website.
This would reduce the risk of having outdated content on either sig-security or the website.

Reply:
Agree. The onboarding.md file will be removed.

Reply:
Oh do you want to remove it completely. I was mainly talking about some of the bullet points. 
The specific linkt to the Security room in Matrix, or the issue process for how to be listed as author is not necessarily covered by the "OSS getting started" section on our website

Reply:
For generic process, I will update the content directly to the documentation under [Tractus-X Eclipse Foundation Rules](https://eclipse-tractusx.github.io/docs/oss/getting-started). 
For specific ones related to security, I will keep only the relevant information under README.md.